durablefunctionsmonitor.dotnetbackend/durablefunctionsmonitor.dotnetbackend.csproj

```diff
@@ -9,6 +9,7 @@
 
   </PropertyGroup>
   <ItemGroup>
+    <PackageReference Include="Azure.Identity" Version="1.10.4" />
```

Comment:
Technically, this dependency comes from Microsoft.Azure.WebJobs.Extensions.DurableTask, and it is them who should care.
We're in contact with the dev team about it.
But until then I agree it makes sense to explicitly lock it here.